FreeRTOS-Plus/Demo/AWS/Jobs_Windows_Simulator/Jobs_Demo/DemoTasks/JobsDemoExample.c

```diff
@@ -968,9 +971,9 @@ void prvJobsDemoTask( void * pvParameters )
             {
                 /* Handler function to process Jobs message payload. */
                 prvNextJobHandler( pxJobMessagePublishInfo );
-                vPortFree( pxJobMessagePublishInfo->pTopicName );
-                vPortFree( pxJobMessagePublishInfo->pPayload );
-                vPortFree( pxJobMessagePublishInfo );
+                vPortFree( ( char * ) ( pxJobMessagePublishInfo->pTopicName ) );
```

Comment:
vPortFree takes a `void *` - https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/portable/MemMang/heap_4.c#L348. Why do we need this cast?

Reply:
`pTopicName` and `pPayload` is declared with const prefix. 

```c
{
    ...
    /**
     * @brief Topic name on which the message is published.
     */
    const char * pTopicName;

    /**
     * @brief Length of topic name.
     */
    uint16_t topicNameLength;

    /**
     * @brief Message payload.
     */
    const void * pPayload;

    /**
     * @brief Message payload length.
     */
    size_t payloadLength;
} MQTTPublishInfo_t;
```
`pxJobMessagePublishInfo` is not. Update in new commit.

Reply:
Shouldn't that cast be to `void *` in that case, to match the API signature?